### ISSUE TITLE
Inline code literal in `.rx` file

### ISSUE THREAD

**Issue by shnewto** [PolySync/howser#12]
I'm unable to fill an Rx in certain cases when using inline code syntax to define some literal content. 

For instance the following combination of `a.rx` and `a.md` report `Rx Rejected!`:

`a.rx`
```
Here is some `code` inline.
```

`a.md`
```
Here is some `code` inline.
```

If the inline code comes at the _end_ of the sentence though, I see `Rx Filled!`. For instance the following combination of `b.rx` and `b.md`

`b.rx`
```
Here is some inline `code`.
```

`b.md`
```
Here is some inline `code`.
```

**Comment by windwardrail** [comment]
Confirmed. Investigating...

**Comment by windwardrail** [comment]
Fixed in #15

**Comment by shnewto** [comment]
i imagine this is a ghost town, closing cause it's associated with me
